artificial-scrum-master/apps/artificial.scrum.master.frontend/src/app/features/estimation-poker/components/add-task-dialog/add-task-dialog.component.ts

```diff
@@ -27,9 +36,11 @@ export class AddTaskDialogComponent {
   private readonly estimationPokerService = inject(EstimationPokerService);
   private readonly toastService = inject(ToastService);
   private readonly translateService = inject(TranslateService);
+  private readonly sessionId: string = inject(MAT_DIALOG_DATA);
 
   #loading = signal<boolean>(false);
-  public loading = this.#loading.asReadonly();
+  #loadingFromService = this.estimationPokerService.isLoading;
```

Comment:
co to za zmienna? troche mało intuicyjnie nazwana i na pierwszy rzut oka nie wiadomo do końca co robi